### PR TITLE
Create category-blog-cn

### PR DESCRIPTION
**Pull request by Nep-Timeline**

### CODE REVIEW

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
LGTM.
@rootmelo92118 what do you think?

**Comment by IceCodeNew** [comment]
suppressed #1658

File: data/category-blog-cn [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# List of all blog domains that have access point in China mainland
+
+huangyuhui.net # 粤ICP备18071565号-1
+owo.today
```
Comment (IceCodeNew) [inline data/category-blog-cn:4]: image

please provide an accessible URL for verification.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/category-blog-cn (hunk 1 above)
Comment (Nep-Timeline) [reply data/category-blog-cn:4]: me.owo.today

Comment (IceCodeNew) [reply data/category-blog-cn:4]: > me.owo.today

这个域名的解析结果指向阿里云香港。和 `category-blog-cn` 分类原则不符。

Comment (IceCodeNew) [reply data/category-blog-cn:4]: 不要擅自把讨论内容标记成已解决。这样很容易漏掉一些还没讨论清楚的问题

**Comment by rootmelo92118** [comment]
> LGTM.
> @rootmelo92118 what do you think?

I think it looks ok.

File: data/category-blog-cn (hunk 1 above)
Comment (Nep-Timeline) [reply data/category-blog-cn]: > 不要擅自把讨论内容标记成已解决。这样很容易漏掉一些还没讨论清楚的问题

好的

**Comment by IceCodeNew** [comment]
> > LGTM.
> > @rootmelo92118 what do you think?
> 
> I think it looks ok.

The last commit should be reverted before we merge this PR.

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**